tools/install_autopkgtest_dependencies.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import subprocess
+import sys
+
+from debian import deb822
+
+
+def main():
+    if len(sys.argv) != 2:
+        sys.exit('Usage: {} test_name'.format(sys.argv[0]))
+    test_name = sys.argv[1]
+    dependencies = get_dependencies(
+        os.path.join('debian', 'tests', 'control'), test_name)
+    subprocess.check_call(['sudo', 'apt', 'install'] + dependencies)
```

Comment:
Can you first check which ones are missing from an installation in order to then run

`sudo apt-mark auto $packages`?